rrbot:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  
  
  # Position Controllers ---------------------------------------
  joint0_velocity_controller:
    type: velocity_controllers/JointVelocityController
    joint: joint0
    pid: {p: 100.0, i: 0.00, d: 10.0}
  joint2_velocity_controller:
    type: velocity_controllers/JointVelocityController
    joint: joint2
    pid: {p: 100.0, i: 0.00, d: 10.0}
  jointPris_velocity_controller:
    type: velocity_controllers/JointVelocityController
    joint: prismatic
    pid: {p: 100.0, i: 0.01, d: 10.0}
